doorway-ui-components/src/global/text.scss

```diff
@@ -146,6 +146,14 @@ h1.title {
   margin-block: var(--text-medium-normal-margin-block, 0 var(--bloom-s3));
 }
 
+.text__medium-weighted {
+  font-family: var(--text-medium-weighted-font-family, var(--bloom-font-alt-sans));
+  font-weight: var(--text-medium-weighted-font-weight, 600);
+  color: var(--text-medium-weighted-color, var(--bloom-color-black));
+  font-size: var(--text-medium-weighted-font-size, var(--bloom-font-size-base));
+  margin-block: var(--text-medium-weighted-margin-block, 0 var(--bloom-s3));
+}
+
```

Comment:
I debated whether using `<b></b>` or adding this in and using it as a className in a span. I started out with `<b>` and then converted. I should standardize on one of them. Thoughts?